deployment/terraform/storage.tf

```diff
@@ -17,7 +17,29 @@ resource "aws_s3_bucket" "static" {
 
 resource "aws_s3_bucket" "storage" {
   bucket = "${lower("${var.environment}")}-pfb-storage-${var.aws_region}"
-  acl    = "private"
+  acl    = "public-read"
+  policy = <<POLICY
```

Comment:
There is an `aws_iam_policy_document` resource that can be used to capture policy definitions:

```hcl
data "aws_iam_policy_document" "read_only_bucket_policy" {
  policy_id = "S3AnonymousReadOnlyPolicy"

  statement {
    sid = "S3ReadOnly"

    effect = "Allow"

    principals {
      type        = "AWS"
      identifiers = ["*"]
    }

    actions = ["s3:GetObject"]

    resources = [
      "arn:aws:s3:::${var.bucket_name}/*",
    ]
  }
}
```

Then `data.aws_iam_policy_document.read_only_bucket_policy.json` can be used to render it where needed.